sdks/python/apache_beam/transforms/stats_test.py

```diff
@@ -156,17 +156,17 @@ def test_get_sample_size_from_est_error(self):
     assert beam.ApproximateUnique._get_sample_size_from_est_error(0.05) == 1600
     assert beam.ApproximateUnique._get_sample_size_from_est_error(0.01) == 40000
 
-  @unittest.skipIf(sys.version_info < (3, 0, 0),
-                   'Skip with py27 because hash function is not good enough.')
-  @retry(reraise=True, stop=stop_after_attempt(3))
+  @unittest.skipIf(sys.version_info < (4, 0, 0),
```

Comment:
Please create a JIRA to re-enable the tests and add it here with a TODO.

Reply:
fixed it, PTAL.